src/PostalRegistry.EventGenerator.CrabPostinfo/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
-  "CrabConnectionString": "data source=.;initial catalog=CRAB;integrated security=True;MultipleActiveResultSets=True;",
+  "CrabConnectionString": "data source=agivsqlcrabbeta;initial catalog=CRAB;integrated security=True;MultipleActiveResultSets=True;",
```

Comment:
Let's keep this one out of the appsettings, even though it is harmless, let's not make it too obvious. Just use `.`